census/core.py

```diff
@@ -315,10 +315,10 @@ def get(self, *args, **kwargs):
 
 class ACS5Client(ACSClient):
 
-    default_year = 2019
+    default_year = 2020
```

Comment:
Changing the default year is causing the tests to fail. Can you either either leave the default year as 2019 and reopen https://github.com/datamade/census/issues/101 (which I can add to my queue), or fix the tests?

Reply:
Hi, @patwater and @sweatercomeback. It would be helpful if you could do one of the two things I proposed in my last:

> Can you either either leave the default year as 2019 and reopen https://github.com/datamade/census/issues/101 (which I can add to my queue), or fix the tests?

Happy to bring this after one of those steps is taken. Let me know what you think.

Reply:
Thanks @hancush ! I believe @sweatercomeback 's PR should address that? Please let us know if not! 

https://github.com/datamade/census/pull/117